content/ros/ROS Introduction/presentation/ROS_Introduction.tex

```diff
@@ -0,0 +1,902 @@
+  \framesubtitle{ROS Concepts}
+    
+ \centering
+ \scalebox{0.9}{\begin{tikzpicture}[sibling distance=9em, level distance = 2.0cm, thick,
+     every node/.style = {shape=rectangle, rounded corners,
+         draw, align=center}]]
+        \node [text width=4cm, fill = yellow]{File system}
+         child { node [text width=2cm, fill = orange!50]{ROS package} 
+        child { node [text width=2cm]{Messages} }
+        child { node [text width=2cm]{Services} }
+        child { node [text width=2cm]{Source code} }
+        child { node [text width=2cm]{Other..} }
+        };
+        \end{tikzpicture}}
+\end{frame}
+
+
+\begin{frame}{File system level}
+    \framesubtitle{ROS Concepts}
+    
+    \centering
+    \scalebox{0.9}{\begin{tikzpicture}[sibling distance=9em, level distance = 2.0cm, thick,
+        every node/.style = {shape=rectangle, rounded corners,
+            draw, align=center}]]
+        \node [text width=4cm, fill = yellow]{File system}
+        child { node [text width=2cm, fill = black!20]{Meta package \\ (stack)} 
+            child { node [text width=2cm, fill = orange!50]{package} }
+            child { node [text width=2cm, fill = orange!50]{package} }
+            child { node [text width=2cm, fill = orange!50]{package} }
+            child { node [text width=2cm, fill = none, draw = none]{..} }
+        };
+        \end{tikzpicture}}
+\end{frame}
+
+
+\begin{frame}{File system level}
+    \framesubtitle{ROS Concepts}
+Inside a ROS package:
+
+\vspace{0.5cm}
+\centering
+\includegraphics[width=.9\linewidth]{figures/example_package.png}
+\end{frame}
+
+    
+\subsection{Computation graph level}
+
+\begin{frame}{Computation graph level}
+    \framesubtitle{ROS Concepts}
+    \begin{itemize}
+        \item In an application that uses ROS, the computations are executed by a collection of processes called Nodes.
+        \item Nodes are connected together in a peer-to-peer network.
+        \item This network of nodes do all the computation and is referred to as ROS computation graph.
+        \item ROS Nodes can be run on single or multiple computers.
+    \end{itemize}
+\end{frame}
+
+\begin{frame}{Computation graph level}
+    \framesubtitle{ROS Concepts}
+    
+    Concepts related to ROS computation graph:
+    
+    \begin{enumerate}
+        \item Nodes.
+        \item Topics.
+        \item Messages.
+        \item Master.
+        \item Services.
+        \item Actions
+        \item Parameter Server.
+        \item Bags.
+    \end{enumerate}
+\end{frame}
+
+\begin{frame}{Computation graph level}
+    \framesubtitle{ROS Concepts}
+    {\huge Nodes:}
+    \vspace{1cm}
+    \begin{itemize}
+        \item A ROS node is a process that exchanges data with other processes through ROS network.
+        
+        \item It may be a python script, a C++ written process, or even a MATLAB script.
+        
+        \item Nodes perform computation.
+    \end{itemize}
+\end{frame}
+
+ \begin{frame}[plain]{}
```

Comment:
Can you please add the sources for the images (either as footnotes or at the end of the slides)? This also holds for the other slides where you have images.

Reply:
Done.
But many of the figures in the slides are my own, they contain logos and other publicly available pictures like the turtle 2 picture, or the PR2 robot, python logo ...etc.